extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/message-interaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - Message interaction details",
+  "description": "Describes interaction made by the profile on the message.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "messageInteraction": {
+      "properties": {
+        "xdm:interactionType": {
+          "title": "Interaction Type",
+          "type": "string",
+          "description": "Specify action made by user on a message.",
+          "enum": ["click", "open", "action"],
+          "meta:enum": {
+            "click": "Link in a message is clicked by the user.",
+            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application.",
+            "action": "Custom action taken on push notifcation, like clicking on some button in push notification."
+          }
+        },
+        "@id": {
+          "title": "Tracking URL ID",
+          "type": "string",
+          "description": "Unique Identifier of the Link clicked by the user."
+        },
+        "xdm:label": {
```

Comment:
Do we need label? is this for user friendly name display in reporting or is there any other reason to carry forward labels?

Reply:
+1 what is this label and where does it come from?

Reply:
> is this for user friendly name display in reporting or is there any other reason to carry forward labels?

Yes this is for reporting use cases in CJA.

> what is this label and where does it come from?

Enrichment service will enrich this using the link id.

Reply:
So this is something that will be set in the authoring environment and that will propagate it to somewhere that enrichment can find it?